mne-cpp.pri

```diff
@@ -45,7 +45,51 @@ defineReplace(MacDeployArgs) {
       deploy_libs_to_copy += $${extra_args}
     }
     return($$deploy_cmd $$deploy_target $$deploy_libs_to_copy)
- }
+}
+
+defineReplace(WinDeployArgs) {
```

Comment:
Just wanna check back if WinDeployArgs is the right choice to name this - and why not Capital letters or all small letters with underscore _